### PR TITLE
Append build folder to output

### PR DESCRIPTION
**Pull request by taylortom**
Fixes: https://github.com/adaptlearning/adapt_authoring/issues/735

### CODE REVIEW

File: plugins/output/adapt/index.js [hunk 1]
```diff
@@ -33,7 +33,9 @@ AdaptOutput.prototype.publish = function (courseId, isPreview, request, response
       tenantId = user.tenant._id,
       outputJson = {},
       isRebuildRequired = false,
+      // should this be Constants.Defaults.ThemeName?
```
Comment (dennis-learningpool) [inline plugins/output/adapt/index.js]: I think you're correct.

File: plugins/output/adapt/index.js [hunk 2]
```diff
@@ -33,7 +33,9 @@ AdaptOutput.prototype.publish = function (courseId, isPreview, request, response
...
       themeName = 'adapt-contrib-vanilla';
+      // should this be Constants.Defaults.MenuName?
```
Comment (dennis-learningpool) [inline plugins/output/adapt/index.js]: ditto

**Comment by dennis-learningpool** [comment]
Hey Tom - I think this is required for Framework 2.0 integration, right? Can you link an issue and update the description? Thanks.

**Comment by taylortom** [comment]
Cool, I'll sort that now.

Another minor question: I've appended the build folder to the output, but is this actually needed? it doesn't look like anything else goes in there atm?
i.e. `path.join(Constants.Folders.AllCourses, tenantId, courseId, Constants.Folders.Build));`
or: `path.join(Constants.Folders.AllCourses, tenantId, courseId));`

**Comment by brian-learningpool** [comment]
I don't think the build folder should be appended to the output, unless something else has changed in the framework?

**Comment by brian-learningpool** [comment]
+1

**Comment by dennis-learningpool** [comment]
+1

**Pull request merged**